blocks/cards/cards.css

```diff
@@ -77,17 +75,25 @@ main .block.cards {
 }
 
 .cards .cards-card-body {
-  width: calc(100% - 1rem - 3.5rem);
   margin: 0 auto;
-  padding: 0 0 0 16px;
   text-align: left;
   color: var(--light-black);
+  padding: 10px 10px 0;
 }
 
 .cards .cards-card-body .title {
   margin-bottom: 1rem;
 }
 
+.cards.large-text .cards-card-body .date {
```

Comment:
Do you see cards where `large-text` may not be needed? If no, we can make this the default style for all the cards.

Reply:
Yes for featured articles font-size is less